client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -48,6 +73,14 @@ export const UserProfilePage = () => {
             </>
           )}
 
+          <ProfileForm
+            loading={loadingUpdate}
+            onSubmit={submitUpdateMe}
+            data={user.name.trim()}
```

Comment:
```suggestion
            data={user}
```
This is more type-safe.  We have to provide `name`, not just any string.  We can trim the data in `submitUpdateMe`